.travis.yml

```diff
@@ -25,3 +25,9 @@ install:
 
 # run a simulation from simulations.py
 script: python -c 'from simdna.simulations import simulate_single_motif_detection; sequences, labels, embedding_arr = simulate_single_motif_detection("SPI1_disc1", 20, 500, 500, 0.4)'
+
+deploy:
+  provider: pypi
+  user: "lilleswing"

```

Comment:
You can use your username/password combo by registering [here](https://pypi.python.org/pypi?%3Aaction=register_form)

And then encrypting your password using the travis CLI
https://docs.travis-ci.com/user/encryption-keys/